www/include/Administration/about/about.php

```diff
@@ -80,7 +80,6 @@
                     <p>Adrien Gelibert</p>
                     <p>Adrien Morais</p>
                     <p>Bruno D'auria</p>
-                    <p>Colin Gagnaire</p>
```

Comment:
Why delete your name ? Move it in the contributors below, you earned it ;)

Reply:
Yes I though about it but I didn't understand how they are organised. I'll put myself at the end of the list.